content/blog/2016/2016-09-30-jenkins-world-2016-wrap-up-community.adoc

```diff
@@ -0,0 +1,75 @@
+---
+layout: post
+title: "Jenkins World 2016 Wrap-up - Community"
+tags:
+- event
+- jenkinsworld
+author: lnewman
+---
+
+NOTE: This is a guest post by link:https://github.com/bitwiseman[Liam Newman],
+Technical Evangelist at link:http://cloudbees.com[CloudBees].
+
+image:/images/post-images/2016-jenkins-world-wrap/jenkins-world-banner-pic.jpg[Jenkins World 2016,height=200,role=right]
+
+This year's link:https://www.cloudbees.com/jenkinsworld/home[Jenkins World] conference
+was a huge milestone for the Jenkins project - the first global event for the Jenkins community.
+It brought users and contributors together to exchange ideas on the current state
+of the project, celebrate accomplishments of the past year, and look ahead at all the exiting enhancements
+coming down the pipe(line).
+
+=== Contributor Summit
+To kick off Jenkins World, we had a full day "Contributor Summit".
+Jenkins is a distributed project with contributors from all over the globe.
+Conferences like this are perfect time to get contributors together face-to-face,
+to talk through current issues and upcoming plans for the the project.
+Some key topic discussed during this summit were:
+
+* *Infrastructure* - In the past year, the Jenkins project has moved to a new website and has entered a
+link:/blog/2016/05/18/announcing-azure-partnership.adoc[partnership with Microsoft Azure]
+to host our infrastructure.
+* *Events* - A year ago, there were five
+link:/projects/jam/[Jenkins Area Meetups], today there are 37 around the
+world, with ~7000 members.
+* *Security* - Daniel Beck has done a great job a "Security Officer" for the project over the last year.
+Jenkins 2 includes tighter security out of the box, 9 security alerts have been addressed, and the
+link:https://jenkins.io/security/[Security Team] is continuing to evaluate threats as they are reported.
+* *Pipeline* - Pipeline has been a success and there's more goodness in progress, including better
```

Comment:
"more goodness in progress" has so many Sses!

what about "more improvements in progress" or something less hissy? :P